modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/api/lifecycle/EditAPIContextAndCheckAccessibilityTestCase.java

```diff
@@ -75,30 +90,48 @@ public void initialize() throws APIManagerIntegrationTestException, XPathExpress
         apiCreationRequestBean.setDescription(API_DESCRIPTION);
         String publisherURLHttp = publisherUrls.getWebAppURLHttp();
         String storeURLHttp = storeUrls.getWebAppURLHttp();
-        apiPublisherClientUser1 = new APIPublisherRestClient(publisherURLHttp);
-        apiStoreClientUser1 = new APIStoreRestClient(storeURLHttp);
 
-        //Login to API Publisher with  admin
-        apiPublisherClientUser1.login(user.getUserName(), user.getPassword());
-
-        //Login to API Store with  admin
-        apiStoreClientUser1.login(user.getUserName(), user.getPassword());
         apiIdentifier = new APIIdentifier(providerName, API_NAME, API_VERSION_1_0_0);
-        apiStoreClientUser1.addApplication(APPLICATION_NAME, APIMIntegrationConstants.RESOURCE_TIER.UNLIMITED, "", "");
+        ApplicationDTO applicationDTO = restAPIStore.addApplication(APPLICATION_NAME,
+                APIMIntegrationConstants.RESOURCE_TIER.UNLIMITED, "", "");
+        applicationId = applicationDTO.getApplicationId();
+        ArrayList grantTypes = new ArrayList();
+        grantTypes.add("client_credentials");
```

Comment:
Shall we use these constants? APIMIntegrationConstants.GRANT_TYPE.CLIENT_CREDENTIAL